.travis.yml

```diff
@@ -1,15 +1,14 @@
 language: android
 
 jdk:
- - oraclejdk7
  - oraclejdk8
 
 android:
   components:
-    - platform-tools
     - tools
```

Comment:
Sorry about the notification spam!
Before this change, the error said there was an old platform-tools version which could not check 25.